tests/llmobs/test_experiments.py

```diff
@@ -325,6 +365,87 @@ def test_dataset_modify_single_record(llmobs, test_dataset, test_dataset_records
     assert ds._version == 2
 
 
+@pytest.mark.parametrize(
+    "test_dataset_records",
+    [[DatasetRecord(input_data={"prompt": "What is the capital of France?"}, expected_output={"answer": "Paris"})]],
+)
+def test_dataset_modify_single_record_empty_record(llmobs, test_dataset, test_dataset_records):
+    assert test_dataset._version == 1
+
+    with pytest.raises(
+        ValueError, match="invalid update, at least one of input_data, expected_output, or metadata must be updated"
+    ):
+        test_dataset.update(
+            0,
+            {},
+        )
```

Comment:
```suggestion
        test_dataset.update(0, {})
```

Reply:
when the world needed `black` the most, it vanished...